packages/components/src/__future__/Tag/_docs/Tag.stickersheet.stories.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react"
+import { Meta, StoryFn } from "@storybook/react"
+import { TagIcon } from "~components/Icons"
+import { StickerSheet } from "~storybook/components/StickerSheet"
+import { Tag } from "../Tag"
+
+type ListProps = {
+  children: React.ReactNode
+}
+const List = ({ children }: ListProps): JSX.Element => (
+  <div className="flex flex-col gap-12 items-start">{children}</div>
+)
+
+export default {
+  title: "KAIO-staging/Tag",
+  parameters: {
+    chromatic: { disable: false },
+    controls: { disable: true },
+  },
+} satisfies Meta
+
+const StickerSheetTemplate: StoryFn = () => (
+  <StickerSheet heading="Tag">
+    <StickerSheet.Header headings={["Label Only", "Icon"]} />
+    <StickerSheet.Body>
+      <StickerSheet.Row>
+        <List>
+          <Tag>Gray</Tag>
+          <Tag color="blue">Blue</Tag>
+          <Tag color="green">Green</Tag>
+          <Tag color="yellow">Yellow</Tag>
+          <Tag color="orange">Orange</Tag>
+          <Tag color="red">Red</Tag>
+          <Tag color="purple">Purple</Tag>
+        </List>
+        <List>
+          <Tag Icon={<TagIcon role="presentation" />}>Gray</Tag>
+          <Tag color="blue" Icon={<TagIcon role="presentation" />}>
```

Comment:
Oof I didn't notice this before, but the icon prop should be lowercase. Don't worry about this being a breaking change, we haven't released it yet 😛